spec/models/spot_spec.rb

```diff
@@ -0,0 +1,37 @@
+# == Schema Information
+#
+# Table name: spots
+#
+#  id         :integer          not null, primary key
+#  title      :string           not null
+#  latitude   :decimal(, )      not null
+#  longitude  :decimal(, )      not null
+#  radius     :decimal(, )      not null
+#  user_id    :integer
+#  topic_id   :integer
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+# Indexes
+#
+#  index_spots_on_topic_id  (topic_id)
+#  index_spots_on_user_id   (user_id)
+#
+
+require 'rails_helper'
+
+describe Spot, type: :model do
+  describe 'validations' do
+    subject(:spot) { build :spot }
+    it { is_expected.to validate_presence_of(:title) }
+    it { is_expected.to validate_presence_of(:latitude) }
+    it { is_expected.to validate_presence_of(:longitude) }
+    it { is_expected.to validate_presence_of(:radius) }
+    it 'is not valid without a user' do
+      expect { create(:spot, user: nil) }.to raise_error(ActiveRecord::RecordInvalid)
```

Comment:
This validation is ok, but I think that is unnecessary since Rails 5 validates `belongs_to` associations.

Add a `describe 'associations'` block and test the `belongs_to` associations there. Here are some available matchers https://github.com/thoughtbot/shoulda-matchers